Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM ruby:3.1
+
+RUN apt update && \
+	apt install -y --no-install-recommends \
+		pipx \
+		postgresql-client && \
+	rm -rf /var/lib/apt/lists/*
+
+WORKDIR /app
+
+RUN /bin/bash -lc "pipx install pre-commit && gem update --system"
+
+ENV PATH="${PATH}:/app/bin"
+
+COPY Gemfile *.gemspec *.lock /app/
+COPY lib/active_record/pg_extensions/version.rb /app/lib/active_record/pg_extensions/
+RUN /bin/bash -lc "bundle install"
```

Comment:
Shouldn't the code be copied in at the very end? (with a `COPY . .`)

I know when you use this with the docker compose stack, there's a bind mount, but the built docker image should probably still be usable independently.

Reply:
I see no reason to. If this was an app you could actually run, definitely. But there's nothing you can do with this gem except develop on it. But if you really feel strongly, I can add it